Cargo.toml

```diff
@@ -4,3 +4,4 @@ version = "0.1.0"
 edition = "2024"
 
 [dependencies]
+lazy-init = "0.5.1"
```

Comment:
lazy-init works around a limitation in Rust where you can't statically initialize a field in a struct declared as Vec<>.  It's a macro that instantiates the structure the first time you access it. So you have to pay a runtime penalty when the program starts, as opposed to having the compiler set it up at compile time.

Reply:
I added it to the main project dependencies because the lazy_init!() macro gets put in the generated code in gen_opcodes.rs